app.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "BadgeYay",
+    "description": "A fully functional badges generator using flask",
+    "repository": "https://github.com/fossasia/badgeyay/",
+    "keywords": [
+        "badgeyay",
+        "fossasia",
+        "flask"
+    ],
+    "buildpacks": [
+        {
+            "url": "heroku/python"
+        }
+    ],
```

Comment:
The comma in the end makes it invalid JSON. Codacy tells this.

Reply:
@niccokunzmann I will rectify it